src/processors/InputProcessor.ts

```diff
@@ -30,7 +30,7 @@ export class InputProcessor {
    * @param input to process
    * @param type of processor to use
    */
-  process(input: any): Promise<CommonInputModel> {
+  process(input: any) : Promise<CommonInputModel> {
```

Comment:
Blank space is necessary? 😄

Reply:
Haha I guess not, but noticed I had it somewhere else 🤷